libnavui-speedlimit/src/main/java/com/mapbox/navigation/ui/speedlimit/view/MapboxSpeedLimitView.kt

```diff
@@ -30,7 +31,7 @@ class MapboxSpeedLimitView : AppCompatTextView {
     private var speedLimitBackgroundColor: Int = 0
     private var speedLimitViennaBorderColor: Int = 0
     private var speedLimitMutcdBorderColor: Int = 0
-    private var speedLimitSign: SpeedLimitSign? = null
+    private var speedLimitSign: SpeedLimitSign? = getSignBasedOnLocale(context)
```

Comment:
The property can be made non-null now, and the null check in `render` is not necessary anymore.

Reply:
+1 to this

Reply:
done